04-brute-force-and-exhaustive-search/1759/hamin.js

```diff
@@ -0,0 +1,43 @@
+const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
+const [L, C] = input.shift().split(' ').map(Number);
+const map = input.join('').split(' ');
+const vowel = ['a', 'e', 'i', 'u', 'o'];
+
+function solution(l, c, arr) {
+
+  const dfs = (s) => {
+    if (s.length === l) {
+      let [vowelCount, consonantCount] = [0, 0];
+
+      for (const m of s) {
+        if (vowel.includes(m)) {
+          vowelCount += 1;
+        } else {
+          consonantCount += 1;
+        };
+      };
+
+      if (vowelCount >= 1 && consonantCount >= 2) {
+        return console.log(s.join(""));
+      };
+
+      return;
```

Comment:
```suggestion
      if (vowelCount >= 1 && consonantCount >= 2) {
        console.log(s.join(""));
      };

      return;
```
콘솔 로그를 리턴하기보다는 그냥 아래서 한 번 리턴하는건 어떨까요?